docs/theory/theory.rst

```diff
@@ -226,14 +235,17 @@ where :math:`\phi` is the randomized phase angle and :math:`N` is the number of
 frequency bands selected to discretize the wave spectrum. For repeatable 
 simulation of an irregular wave field :math:`S(\omega)`, WEC-Sim allows 
 specification of :math:`\phi`, refer to the :ref:`user-advanced-features-seeded-phase` 
-section. Additionally, an excitation force impulse response function is defined 
+section. 
+
+Additionally, an excitation force impulse response function is defined 
 as 
 
 .. math::
 
     K_{e}(t) = \frac{1}{2\pi} \intop_{0-\infty}^{\infty}
```

Comment:
@akeeste In the lower bound of the integral it should just be -\infty

Reply:
good catch, I will update this